lib/que/job.bulk_enqueue_spec.rb

```diff
@@ -0,0 +1,383 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe Que::Job, '.bulk_enqueue' do
```

Comment:
I'd like to see a few more test cases, e.g.:

- When there are no args
- When there are no kwargs
- With various job options
- With synchronous mode

Reply:
Yep, will do heaps more, not ready for full review yet.

Reply:
Yep, just thought I'd note my thoughts ;)